client/public/locales/en/common.json

```diff
@@ -150,6 +151,7 @@
             "archived": "Archived"
         },
         "fields_help": {
+            "price": "Price of a full spool in the system configured currency.",
```

Comment:
Here you should add that if this is not set, the filament's price will be assumed instead.

Reply:
Added as requested.